src/styles/stars.scss

```diff
@@ -0,0 +1,13 @@
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+
+  width: 16px;
+  height: 16px;
+
+  &:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
Consider adding styles for every modifier from the previous task

Reply:
I cant understand what i need to do here.

Reply:
Read this requirement point:
![image](https://github.com/mate-academy/layout_product-cards/assets/64101898/6823f603-e792-4053-b42e-5cc50d4bd3a2)
It should be operational so just use it like in the previous task with stars, you should be able to change amount of stars from html modifier

Reply:
https://github.com/mate-academy/layout_product-cards/pull/3538/files#r1311914703

check your stars task you already did it
https://github.com/mate-academy/layout_stars/pull/3398/files